florist/tests/integration/api/routes/server/test_job.py

```diff
@@ -90,3 +90,245 @@ async def test_new_job_fail_bad_server_info(mock_request) -> None:
 
     assert exception_info.value.status_code == 400
     assert "job.server_info could not be parsed into JSON" in exception_info.value.detail
+
+async def test_list_jobs_with_status(mock_request) -> None:
```

Comment:
Nice tests! 👏

Reply:
Going through your PRs has been helpful to learn :) mock_request fixture you made is nifty